wagtail/admin/models.py

```diff
@@ -63,7 +63,7 @@ def get_object_usage(obj):
 
 def popular_tags_for_model(model, count=10):
     """Return a queryset of the most frequently used tags used on this model class"""
-    content_type = ContentType.objects.get_for_model(model)
+    content_type = ContentType.objects.get_for_model(model, for_concrete_model=False)
```

Comment:
@gasman I'm not 100% sure this is correct. I think it might actually be better the other way around, but I'm not sure the entries are stored that way.

Reply:
Answer: Item models use GenericForeignKey with the default `for_concrete_model` value of `True`, which means this should change to match:
https://github.com/jazzband/django-taggit/blob/master/taggit/models.py#L144

Reply:
Fixed in [57e74c0](https://github.com/wagtail/wagtail/pull/9255/commits/57e74c0a2686204b31f900c1a3ea412326cd736b)